src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,14 +1,31 @@
 package core.basesyntax;
 
-/**
- * <p>Дано файл, потрібно прочитати його вміст і вибрати всі слова що починаються на `w`.
- * Результат повернути у вигляді відсортованого масиву (за замовчуванням). Всі слова повинні
- * бути в нижньому регістрі. У випадку якщо таких слів не знайдено повернути пустий масив.
- * Приклад: Width world Wide web
- * Результат: web wide width world</p>
- */
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 public class FileWork {
+    private static final String LETTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        try {
+            List<String> preResult = new ArrayList<String>();
+            String file = Files.readString(Paths.get(fileName)).toLowerCase();
+            if (file.length() > 0) {
+                for (String word : file.split("[^A-Za-zА-Яа-я]+")) {
+                    if (word.substring(0, 1).equals(LETTER)) {
```

Comment:
there is a method in class String that checks if a String starts with a certain value

Reply:
done